components/filter/test/filter.test.js

```diff
@@ -433,8 +433,10 @@ describe('d2l-filter', () => {
 
 			const newDim = document.createElement('d2l-filter-dimension-set');
 			newDim.key = 'newDim';
+			newDim.text = 'New Dim';
```

Comment:
Unrelated, this fixes come console warnings after @dbatiste  added the LabelMixin stuff